ServiceAPIExtensions/Controllers/ContentAPiController.cs

```diff
@@ -154,7 +155,18 @@ ContentReference FindContentReference(string reference)
             return result;
         }
 
-        private static string GetContentType(IContent c)
+        private string GetContentType(IContent c)
+        {
+            if (_typerepo.Load(c.GetOriginalType().Name) != null)

```

Comment:
This suggested to me that we don't use the correct way to fetch the ContentType. I looked into it and it seems we can do it correctly with:

`return _typerepo.Load(c.ContentTypeID)?.Name`

I think we can drop the `?` also, since at this point the ContentTypeID should always exist.